.github/workflows/merge.yml

```diff
@@ -0,0 +1,27 @@
+name: merge
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        # Do the ones not covered in PR workflow
+        os: [ubuntu, macos, PyPy]
+        python-version: ["3.10"]  # , "3.12" except PyPy
+        test-extras: [false]
```

Comment:
wouldn't it make sense to test extras for macos at least once?

Reply:
That's a good point... I'll see about adding that.  I was also wondering if there were anything we could push to the 'publish' workflow for pydicom releases, but this seems important enough to find out about earlier.